onnx/backend/test/case/node/splittosequence.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import onnx
+
+from ..base import Base
+from . import expect
+
+
+class SplitToSequence(Base):
+    @staticmethod
+    def export_with_split_1() -> None:
+        data = np.arange(18).reshape((3, 6)).astype(np.float32)
+        split = np.array(2, dtype=np.int64)
+
+        node = onnx.helper.make_node(
+            "SplitToSequence", ["data", "split"], ["seq"], axis=1
+        )
+
+        expected_outputs = [
+            [
+                np.array([[0.0, 1.0], [6.0, 7.0], [12.0, 13.0]], dtype=np.float32),
```

Comment:
So I think one difference with the backend test and torch's OpInfo is the effort required to create one. OpInfo is essentially a dataclass with information about specific ops, including example inputs. This makes it very easy to add test inputs, typically as a matter of 2 lines.

Reply:
As an example, the input samples for split: https://github.com/pytorch/pytorch/blob/23d58fedb107181b90725a72b97c5ea184252c93/torch/testing/_internal/common_methods_invocations.py#L162-L183

Negative tests: https://github.com/pytorch/pytorch/blob/23d58fedb107181b90725a72b97c5ea184252c93/torch/testing/_internal/common_methods_invocations.py#L221-L236

And the OpInfo: https://github.com/pytorch/pytorch/blob/23d58fedb107181b90725a72b97c5ea184252c93/torch/testing/_internal/common_methods_invocations.py#L13640-L13648

Reply:
I think onnx is close to that. The only thing I find annoying is the the generation of the documentation, test case and stat coverage before adding the changes. That may be the topic of another PR.

Reply:
Yes, it would be great to have automatic generation (of docs etc.) happen fully automatically in the backend, but that's a different topic.